VisualizationBase/src/nodes/ViewItemNode.cpp

```diff
@@ -27,26 +27,13 @@
 #include "ViewItemNode.h"
 
 #include "ModelBase/src/nodes/TypedListDefinition.h"
-DEFINE_TYPED_LIST(Visualization::ViewItemNode)
 
 namespace Visualization {
 
-NODE_DEFINE_TYPE_REGISTRATION_METHODS(ViewItemNode)
-
-ViewItemNode::ViewItemNode(Model::Node* parent)
```

Comment:
I think once you bring back the few macro calls that you removed, you'll need the standard constructors again.